docs/src/pages/careers/lead-designer.md

```diff
@@ -0,0 +1,89 @@
+We are building [a low-code platform](https://mui.com/toolpad/) for developers to significantly increase the speed and lower the barrier for creating simple CRUD/dashboard UIs with MUI.
+
+We are a fully distributed team, spread across multiple time zones.
+We work mainly asynchronously, relying mostly on written collaboration.
+Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
+For additional details about the MUI team and culture, you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+Both our open-source community and our premium products are growing fast (x2 YoY).
+We need talented people to keep that going!
+
+We are aiming to make high-end design accessible to the many, empowering developers that lack access to a designer as well as saving time for designers with a source they can trust.
+[According to our last survey](https://mui.com/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library), the quality of the design execution (UX & UI) is the most important criteria used by developers & designers to pick their next UI library.
+
+### Why this is interesting
+
+Our solution empowers React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+
+But while we are [the leading](https://tsh.io/state-of-frontend/#over-the-past-year-which-of-the-following-design-systems-was-your-favorite-go-to-solution) UI design system in the frontend space, the adoption of MUI is only at 25%. More importantly, the challenges of developers and designers to solve when creating UIs go way beyond the ones of design systems.
+We envision a future where MUI becomes the default toolkit for web developers to create UIs.
+It's why we started MUI Base, Joy UI, and MUI Toolpad. Design is key to achieving this goal.
+
+## About the role
+
+### What you'll do on a day-to-day basis
+
+Depending on the day, you'll:
+
+- Make the product experience more delightful, e.g. visually more appealing
+- Pay attention to all the smallest interface interactions so they yield an outstanding UX
+- Develop the brand
+- Refresh and expand the marketing pages, e.g. with a showcase & case study section
+- Design the next features across the different product
+- Shape the direction of the design kit products
+- **Solve organization issues**
+  - You will improve how the design responsibilities are covered.
+  - You will ensure the consistency of design and product direction throughout all the products and all customer-facing artifacts.
+- **Exercise team leadership**
+  - You will overseas the overall design quality
+  - You will manage designer's career
+  - You will hire and maintain a great team
```

Comment:
- Ensure high quality UI and UX across teams, libraries, components, and product strategy
- Collaborate with product management and engineering to ensure that design work is aligned with business goals and user needs
- Create design concepts, prototypes, and high-fidelity mockups for projects on the X, Core, and Toolpad teams
- Update and implement new components on the Core, X, and Toolpad teams
- Develop and maintain MUI’s design system and style guide

### As a team leader, you will:

- Manage the design process, including user research, user flows, and user testing
- Review design work and provide feedback to ensure overall design quality and consistency, with consideration to both UI and UX.
- Set goals, priorities, and distribute tasks for the design team
- Provide mentorship and support to two existing design team members
- Hire and retain a great team
- Oversee the Design Kits, ensuring they’re aligned with engineering implementation.